core/models/DbModel.php

```diff
@@ -32,11 +32,32 @@ public function save()
         return true;
     }
 
+    public function update($value, $where)
+    {
+        $tablename = static::tableName();
+        $attrWhere = array_keys($where);
+        $attrValue = array_keys($value);
+
+        $sqlWhere = implode("AND ", array_map(fn ($attr) => "$attr = :$attr ", $attrWhere));
+        $sqlValue = implode(", ", array_map(fn ($attr) => "$attr = :$attr", $attrValue));
+
+        $statement = self::prepare("UPDATE $tablename SET $sqlValue WHERE $sqlWhere");
+
+        foreach ($value as $key => $item) {
+            $statement->bindValue(":$key", $item);
+        }
+        foreach ($where as $key => $item) {
+            $statement->bindValue(":$key", $item);
+        }
```

Comment:
I modified the keys so that the `$sqlWhere` will now have `_` infront.
```sql
UPDATE table SET id = :id WHERE id = :_id;
```
This should ensure that each binding will have different keys' syntax to look for.

Reply:
That should work, though I would recommend in the future to use something more descriptive. Eventually, you will hvae to debug these things. Remembering what the `_` means in those settings will make it harder to debug incorrect queries.